src/presentation/react-components/BulkDataImport.tsx

```diff
@@ -0,0 +1,78 @@
+import Title from "antd/lib/typography/Title";
+import { useCallback, useState } from "react";
+import { Alert, Breadcrumb, Button, Space, Statistic } from "antd";
+import Paragraph from "antd/lib/typography/Paragraph";
+import Link from "next/link";
+import {
+  buildBulkDataImportResultsViewModel,
+  BulkDataImportViewModel,
+} from "../BulkDataImport";
+import { performBulkDataImport } from "../../api-client/bulkDataImport";
+
+const BulkDataImport = ({ readingCount }: BulkDataImportViewModel) => {
+  const [isInProgress, setIsInProgress] = useState(false);
+  const [successMessage, setSuccessMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const performImport = useCallback(async () => {
+    setIsInProgress(true);
+    setErrorMessage("");
+    setSuccessMessage("");
+
+    const resp = await performBulkDataImport();
+    const { message, isGood } = buildBulkDataImportResultsViewModel(resp);
+
+    if (isGood) setSuccessMessage(message);
+    else setErrorMessage(message);
```

Comment:
Did eslint not complain about no curly braces here? I know it's valid code, but I don't like it. Just like I always want semi colons in my JS, even though they're not technically required 😬

Reply:
I think it doesn't mind as long as it's a single line.